doc/classes/Color.xml

```diff
@@ -19,123 +19,17 @@
 		<method name="Color">
 			<return type="Color">
 			</return>
-			<argument index="0" name="from" type="String">
-			</argument>
-			<description>
-				Constructs a color from an HTML hexadecimal color string in RGB or RGBA format. See also [method @GDScript.ColorN].
-				[codeblocks]
-				[gdscript]
-				# Each of the following creates the same color RGBA(178, 217, 10, 255).
-				var c3 = Color("#b2d90a") # RGB format with "#".
-				var c4 = Color("b2d90a") # RGB format.
-				var c1 = Color("#b2d90aff") # RGBA format with "#".
-				var c2 = Color("b2d90aff") # RGBA format.
-				[/gdscript]
-				[csharp]
-				// Each of the following creates the same color RGBA(178, 217, 10, 255).
-				var c3 = new Color("#b2d90a");
-				var c4 = new Color("b2d90a"); // RGB format.
-				var c1 = new Color("#b2d90aff");
-				var c2 = new Color("b2d90aff"); // RGBA format.
-				[/csharp]
-				[/codeblocks]
-				You can also use the "web color" short-hand form by only using 3 or 4 digits.
-				[codeblocks]
-				[gdscript]
-				# Each of the following creates the same color RGBA(17, 34, 51, 255).
-				var c3 = Color("#123") # RGB format with "#".
-				var c4 = Color("123") # RGB format.
-				var c1 = Color("#123f") # RGBA format with "#".
-				var c2 = Color("123f") # RGBA format.
-				[/gdscript]
-				[csharp]
-				// Each of the following creates the same color RGBA(17, 34, 51, 255).
-				var c3 = new Color("#123");
-				var c4 = new Color("123"); // RGB format.
-				var c1 = new Color("#123f");
-				var c2 = new Color("123f"); // RGBA format.
-				[/csharp]
-				[/codeblocks]
-			</description>
-		</method>
-		<method name="Color">
-			<return type="Color">
-			</return>
-			<argument index="0" name="from" type="int">
-			</argument>
-			<description>
-				Constructs a color from a 32-bit integer (each byte represents a component of the RGBA profile).
-				[codeblocks]
-				[gdscript]
-				var c = Color(274) # Equivalent to RGBA(0, 0, 1, 18)
-				[/gdscript]
-				[csharp]
-				var c = new Color(274); // Equivalent to RGBA(0, 0, 1, 18)
-				[/csharp]
-				[/codeblocks]
```

Comment:
This needs to be bound again as a `Color::Color(int)` constructor that calls `Color::hex()`.